contents/handbook/engineering/ee-setup.md

```diff
@@ -4,23 +4,31 @@ sidebar: Handbook
 showTitle: true
 ---
 
-Rough instructions on how to "get the EE version running in all its glory".
+Rough instructions on how to "get the EE version running in all its glory". **Currently for internal use only.**
 
-Currently for internal use only.
 
-### Getting it up and running
-`docker-compose -f ee/docker-compose.ch.yml up`
+There are multiple ways to run your EE setup depending on your preferences and needs.
 
-### Fixing broken frontend build
-`docker run ee_web yarn build`
+### Option A (Recommended). Docker-based.
+Use our handy command prebuilt command to run. Everything will run on Docker. Frontend will be built and KEA typegen will be run on the background so you can work on the frontend too.
 
-### Running Backend in Docker and Frontend locally
+```
+yarn start-ch-dev
+```
+
+If you're running into problems, such as Kafka connection errors, frontend not building, etc. we recommend starting fresh by cleaning your Docker installation.
+
+```
+docker system prune -a
```

Comment:
This does not prune running things.

A better way is to `docker-compose stop` and `docker-compose rm -v` (if needed add the appropriate dockerfile)